fbw-common/src/wasm/systems/systems/src/hydraulic/flap_slat.rs

```diff
@@ -121,17 +123,97 @@ impl Actuator for FlapSlatHydraulicMotor {
     }
 }
 
-pub struct FlapSlatAssembly {
-    position_left_percent_id: VariableIdentifier,
-    position_right_percent_id: VariableIdentifier,
+pub struct SecondarySurface {
```

Comment:
I don't quite get why this is called `SecondarySurface` if it incorporates more than 1 surface? Maybe you find a better name?

Reply:
I wanted to avoid using `FlapsSlatsSurface` and give the struct a more generic name. `SecondarySurface` comes from secondary control surfaces which includes flaps/slats.